HeRoBackEnd/Controllers/UserController.cs

```diff
@@ -1,25 +1,62 @@
-﻿using HeRoBackEnd.ViewModels;
+﻿using Data.Entities;
+using HeRoBackEnd.ViewModels.User;
 using Microsoft.AspNetCore.Mvc;
+using PagedList;
 using Services.Services;
-//using Data.Entities;
 
 namespace HeRoBackEnd.Controllers
 {
     public class UserController : Controller
     {
-        //private UserServices userService;
+        public IUserService userService;
 
-        public UserController()
+        public UserController(IUserService _userService)
         {
-            //UserServices userService = new UserServices();
+            userService = _userService;
         }
 
-        public IActionResult Index()
+        public IActionResult GetList(PagingViewModel pagingViewModel)
         {
-            //List<User> users = usersService.GetAllActive();
-            
-            //return new JsonResult(users);
-            return View();
+            IEnumerable<User> users = userService.GetUsers();
```

Comment:
All logic should be in the service. Controller should only prepare DTOs and execute service methods